msPackage.json

```diff
@@ -12,6 +12,9 @@
     "davidanson.vscode-markdownlint",
     "ecmel.vscode-html-css",
     "pranaygp.vscode-css-peek",
-    "patbenatar.advanced-new-file"
+    "patbenatar.advanced-new-file",
+    "redwan-hossain.auto-rename-tag-clone",
```

Comment:
This plugin don't exist in the [Microsoft marketplace](https://marketplace.visualstudio.com) here we planed once to add Microsoft specific add-ons. Sadly Its currently not working in our pipeline.

Reply:
Hab die von OpenVSX genommen.
Wusste bloss nicht ob die da die vom Marketplace oder die von OpenVSX eintragen soll.
Es gibt alle bis auf das JS Addon mittlerweile dort :D